docs/source/educational-resources.rst

```diff
@@ -0,0 +1,12 @@
+Educational Resources
+=======
```

Comment:
I think this needs to match the length of the text.

Reply:
Sorry, my bad. I'll fix it.